tools/trace_analyzer_tool.h

```diff
@@ -164,8 +165,89 @@ struct CfUnit {
   std::map<uint32_t, uint32_t> cf_qps;
 };
 
+class TraceAnalysisResult : public TraceRecordResult {
```

Comment:
So, each query type will have a TraceAnalysisResult to hold all the trace results? What if the trace size is TB level, we cannot hold them in memory?